apps/docs/pages/guides/auth/auth-user-management.mdx

```diff
@@ -0,0 +1,130 @@
+        "full_name": "Foo Bar",
+        "iss": "https://projectref.supabase.com",
+        "name": "Foo",
+        "picture": "",
+        "sub": "000000000000000000000"
+      },
+      "provider": "google",
+      "last_sign_in_at": "2023-11-03T16:36:32.247986-04:00", // the time that the identity was last used to sign in 
+      "created_at": "2023-11-03T16:36:32.248001-04:00", // the time that the identity was created
+      "updated_at": "2023-11-06T17:02:38.271966-05:00" // the time that the identity was last updated
+    },
+    {
+      "id": "00000000-0000-0000-0000-000000000000",
+      "user_id": "00000000-0000-0000-0000-000000000000",
+      "identity_data": {
+        "email": "foo@example.com",
+        "sub": "00000000-0000-0000-0000-000000000000"
+      },
+      "provider": "email",
+      "last_sign_in_at": "2023-11-03T16:42:50.928343-04:00",
+      "created_at": "2023-11-03T16:42:50.928367-04:00", 
+      "updated_at": "2023-11-03T16:42:50.928367-04:00"
+    }
+  ],
+  "created_at": "2023-11-03T16:36:32.238822-04:00", // the time that the user signed up 
+  "updated_at": "2023-11-06T17:02:38.295236-05:00" // the time that the user object was last updated
+}
+```
+
+## Configuration
+Supabase Auth provides these [configuration options](/dashboard/project/_/settings/auth) to control user access to your application:
+- **Allow new users to sign up**: Users will be able to sign up. If this config is disabled, only existing users can sign in.
+- **Allow unverified email sign in**: Users will not need to have a verified email address to sign in.
+    - If enabled, you can structure your RLS policies to provide different access controls to a user with an unverified email and a user with a verified email. For example, 
+    ```sql
+    -- Allows a user to read all posts regardless of whether they have a verified email address
+    create policy "policy_name"
+    ON public.posts
+    for select to authenticated using (
+        true
+    );
+
+    -- Only allow users with a verified email address to insert posts
+    create policy "policy_name"
+    ON public.posts
+    for insert to authenticated with check (
+        auth.jwt()->>'email_verified' is true
+    );
+    ```
+- **Confirm Email (Deprecated)**: Users will need to confirm their email address before signing in for the first time.
+    - Having **Confirm Email** disabled assumes that the user's email does not need to be verified in order to login and implicitly confirms the user's email in the database.
+    - If you previously relied on this config to autoconfirm a user's email address, you can switch to use **Allow unverified email sign in** instead. This new option allows the user to sign in with an unverified email which you can keep track of through the user object. It provides more versatility if you require your users to verify their email address in the future since you can structure your RLS policies to check the user's `email_verified` field.
```

Comment:
Can consider adding this as a separate header  (e.g. ### Migrating from `Confirm Email` to  **Allow unverified email sign in** ) for better visibility.